lib/whitehall_importer/integrity_checker/unpublishing_check.rb

```diff
@@ -0,0 +1,59 @@
+module WhitehallImporter
+  class IntegrityChecker::UnpublishingCheck
+    attr_reader :edition, :publishing_api_unpublishing
+
+    def initialize(edition, publishing_api_unpublishing)
+      @edition = edition
+      @publishing_api_unpublishing = publishing_api_unpublishing
+    end
```

Comment:
Minor - since we don't have a concept of unpublishing in content publisher you might be able to shorten this to just unpublishing throughout the class.

Reply:
👍

Reply:
Done